addons/account/tests/test_account_move_reconcile.py

```diff
@@ -179,6 +179,8 @@ def assertAmountsGroupByAccount(self, amount_per_account):
         if not expected_values:
             return
 
+        self.env['account.move'].flush()
```

Comment:
This will flush everything. You should only flush `account.move.line`

```suggestion
        self.env['account.move.line'].flush(self.env['account.move.line']._fields)

Reply:
isn't the same without explicit fields?

```suggestion
        self.env['account.move.line'].flush()
```

Reply:
Nope 
```python
    @api.model
    def flush(self, fnames=None, records=None):
        """ Process all the pending computations (on all models), and flush all
        the pending updates to the database.

        :param fnames (list<str>): list of field names to flush.  If given,
            limit the processing to the given fields of the current model.
        :param records (Model): if given (together with ``fnames``), limit the
            processing to the given records.
        """
```